RELEASE_NOTES.md

```diff
@@ -19,7 +19,7 @@
 
 -->
 
-# Apache IoTDB (incubating) 0.8.0
+# Apache IoTDB (incubating) 0.9.0
 
 This is the first official release of Apache IoTDB after joining the Incubator.
```

Comment:
I would expect the notes to contain all history notes, not just the latest ones. In other words, I would prefer to see release notes of 0.8.0, 0.8.1 and 0.9.0 appear on the same page.
With that said, "This is the first official release of Apache IoTDB after joining the Incubator." is talking about 0.8.0 :)

Reply:
fixed